src/main/java/org/gitlab4j/api/GroupApi.java

```diff
@@ -224,6 +225,32 @@ public GroupApi(GitLabApi gitLabApi) {
         return (new Pager<Group>(this, Group.class, itemsPerPage, formData.asMap(), "groups", groupId, "subgroups"));
     }
 
+    /**
+     * Get a list of projects belonging to the specified group ID.
+     *
+     * GET /groups/:id/projects
+     *
+     * @param groupId the group ID to list the projects for
+     * @param archived Limit by archived status
+     * @param visibility Limit by visibility public, internal, or private 
+     * @param orderBy Return projects ordered by id, name, path, created_at, updated_at, or last_activity_at fields. Default is created_at
+     * @param sortOrder Return projects sorted in asc or desc order. Default is desc
+     * @param search Return list of authorized projects matching the search criteria
+     * @param simple Return only the ID, URL, name, and path of each project
+     * @param owned Limit by projects owned by the current user
+     * @param starred Limit by projects starred by the current user
+     * @param withIssuesEnabled Limit by enabled issues feature
+     * @param withMergeRequestsEnabled Limit by enabled merge requests feature
+     * @param withCustomAttributes Include custom attributes in response (admins only)
+     * @return a list of projects belonging to the specified group ID
+     * @throws GitLabApiException if any exception occurs
+     */
+  public List<Project> getProjects(int groupId, ProjectOfGroupFilter filter) throws GitLabApiException {
```

Comment:
For new methods that take a group ID this library is now using ```Object groupIdOrPath``` instead of ```int groupId```.
You then use ```getGroupIdOrPath()``` like this:
```
    Response response = get(Response.Status.OK, formData.asMap(), "groups",  getGroupIdOrPath(groupIdOrPath), "projects");
    return (response.readEntity(new GenericType<List<Project>>() {}));
```

Reply:
Adjusted. 👍